### ISSUE TITLE
Injection data exception

### ISSUE THREAD

**Issue by YMY1666527646** [KillSKID/Genshin-Cheetos#3]
Hey, there was no abnormality in downloading and injecting data before. I can find it now
After the new offset is changed, the data exception is injected, but I decompress the file again and build the DLL. The injection is the same. Has the data exception been detected?
Can this problem be solved?

**Comment by KillSKID** [comment]
Outdated address so you can't inject this cheet in gensihn2.3

**Comment by YMY1666527646** [comment]
> Outdated address so you can't inject this cheet in gensihn2.3

No, I have updated the cheating address in 2.3, but the injection is still a data exception

**Comment by KillSKID** [comment]
It wont happen

**Comment by anotherfinemess84** [comment]
are you updating to 2.4?
